src/custom/components/Header/index.tsx

```diff
@@ -185,6 +190,7 @@ export default function Header() {
           <Menu />
         </HeaderElementWrap>
       </HeaderControls>
+      <OrdersPanel ordersPanelOpen={ordersPanelOpen} closeOrdersPanel={closeOrdersPanel} />
```

Comment:
nit: can you rename these to be more aligned? e.g `ordersPanelOpen` and `ordersPanelClose` or `openOrdersPanel` and `closeOrdersPanel`

Reply:
ah i see now this is the boolean indicator type. in that case can you rename to sth more clear like: `isPanelOpen` or simply `isOpen`

Reply:
second that nit